### PR TITLE
feat-auth-pages

### PR DESCRIPTION
**Pull request by ankit9015**
This PR has following features:
1. Static page for Login and Signup
2. Login functionality added
3. Remember user of login form saves the login information

-- Instruction: Go to Profile page and then go to Login in the deploy preview

### CODE REVIEW

File: src/components/PasswordInput/PasswordInput.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          placeholder="******"
+          required
+          value={_value}
+          onChange={_onChange}
+        ></input>
+        <span
+          className="text-blue password-show"
+          onClick={() => setShowPassword((prev) => !prev)}
+        >
+          {showPassword ? <FaEye /> : <FaEyeSlash />}
```
Comment (SaurabhChawda) [inline src/components/PasswordInput/PasswordInput.jsx:24]: Liked that you are using Component for show/hide password Icon.

File: src/context/AuthContext/AuthContext.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import { loginHandlerService } from "../../service/authentication/loginHandlerService";
+
+const AuthContext = createContext({});
+
+const AuthProvider = ({ children }) => {
+  const authToken = JSON.parse(localStorage.getItem("AUTH-TOKEN"));
+
+  const navigate = useNavigate();
+  const location = useLocation();
+  console.log(location);
```
Comment (SaurabhChawda) [inline src/context/AuthContext/AuthContext.js:12]: Remove Console if no need

File: src/page-layout/Main/Main.jsx [hunk 3]
```diff
@@ -21,10 +24,19 @@ function Main() {
           <Route path="/" element={<Home />} />
           <Route path="/home" element={<Home />} />
```
Comment (SaurabhChawda) [inline src/page-layout/Main/Main.jsx:25]: is it good to use two route for same page ?

**Review by SaurabhChawda — APPROVED** [review]

File: src/context/AuthContext/AuthContext.js (hunk 2 above)
Comment (ankit9015) [reply src/context/AuthContext/AuthContext.js:12]: It is removed in the local file, don't know why it didn't update in github

File: src/page-layout/Main/Main.jsx (hunk 3 above)
Comment (ankit9015) [reply src/page-layout/Main/Main.jsx:25]: The root route is for landing page, will update it

File: src/pages/MockMan/MockMan.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import Mockman from "mockman-js";
+function MockAPI() {
+  return (
+    <div className="MockAPI">
```
Comment (rushikesh3299) [inline src/pages/MockMan/MockMan.js:5]: No need to create this extra file for mockman i guess you can directly 
import Mockman from "mockman-js"; & use as <Route path="/mockman" element={<Mockman />} />
where it is requird

**Review by rushikesh3299 — APPROVED** [review]
Code is well structured

File: src/pages/MockMan/MockMan.js (hunk 4 above)
Comment (ankit9015) [reply src/pages/MockMan/MockMan.js:5]: Thanks, will try that for next project

**Pull request merged**